src/main.ts

```diff
@@ -41,6 +42,13 @@ async function installGNATCommunity (year : string, target : string) {
     var pack = release.pack;
     var url = release.url;
     var sha1 = release.sha1;
+    const bin = path.resolve(cache, 'bin');
```

Comment:
Cached path should still include the package name (`pack`). Otherwise multiple installs will be in the same folder and it won't be possible to detect if the cache install is indeed the one requested (native, arm-elf, or riscv32-elf).